index.html

```diff
@@ -18,12 +18,20 @@
     <div class="header-content row">
       <div class="logo"><a href="/">Welcome</a></div>
       <nav class="nav">
-        <svg aria-label="Menu Icon" class="menu-icon" width="18" height="12" viewBox="0 0 18 12"
-          xmlns="http://www.w3.org/2000/svg">
-          <path fill-rule="evenodd" clip-rule="evenodd"
-            d="M0 1a1 1 0 0 1 1-1h16a1 1 0 1 1 0 2H1a1 1 0 0 1-1-1zm0 5a1 1 0 0 1 1-1h16a1 1 0 1 1 0 2H1a1 1 0 0 1-1-1zm1 4a1 1 0 1 0 0 2h16a1 1 0 1 0 0-2H1z" />
-        </svg>
-        <ul class="menu-list">
+        <div class="menu-icons-container margin-left">
+          <svg aria-label="Menu Icon" class="menu-icon menu-icon-open " width="18" height="12" viewBox="0 0 18 12"
+            xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd"
+              d="M0 1a1 1 0 0 1 1-1h16a1 1 0 1 1 0 2H1a1 1 0 0 1-1-1zm0 5a1 1 0 0 1 1-1h16a1 1 0 1 1 0 2H1a1 1 0 0 1-1-1zm1 4a1 1 0 1 0 0 2h16a1 1 0 1 0 0-2H1z" />
+          </svg>
+          <svg class="menu-icon menu-icon-close hidden" width="14" height="14" viewBox="0 0 14 14" fill="none"
+            xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd"
+              d="M0.292893 0.292893C0.683417 -0.0976311 1.31658 -0.0976311 1.70711 0.292893L7 5.58579L12.2929 0.292893C12.6834 -0.0976311 13.3166 -0.0976311 13.7071 0.292893C14.0976 0.683417 14.0976 1.31658 13.7071 1.70711L8.41421 7L13.7071 12.2929C14.0976 12.6834 14.0976 13.3166 13.7071 13.7071C13.3166 14.0976 12.6834 14.0976 12.2929 13.7071L7 8.41421L1.70711 13.7071C1.31658 14.0976 0.683417 14.0976 0.292893 13.7071C-0.0976311 13.3166 -0.0976311 12.6834 0.292893 12.2929L5.58579 7L0.292893 1.70711C-0.0976311 1.31658 -0.0976311 0.683417 0.292893 0.292893Z" />
+          </svg>
+        </div>
```

Comment:
- [ ] You did a great job on the functionalities but you need to work on the design, Please fix the below issues and make sure your design matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.
  - Please make sure the drop-down menu contains 100% of the width.
  - Please reduce the left-margin for the mobile menu.
  - Please increase the font-size for the mobile menu options.
  - Please use the correct color for the close(X) button.

### Your design
![image](https://user-images.githubusercontent.com/83803414/194929828-3dc81bd3-91f0-4df5-b0d7-1b38e511af63.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/194929887-8618859f-7cb2-478b-82af-3e0c6430f918.png)